mindjunkies/home/views.py

```
@@ -2,109 +2,96 @@
 from django.shortcuts import render
 from django.utils.safestring import mark_safe
 from django.views.decorators.http import require_http_methods
+from django.views import View
 
 from mindjunkies.courses.models import Course, CourseCategory, Enrollment, LastVisitedCourse
```

Ruff (0.8.2)

`mindjunkies.courses.models.LastVisitedCourse` imported but unused

Remove unused import: `mindjunkies.courses.models.LastVisitedCourse`

(F401)